hangman.js

```diff
@@ -1 +1,44 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word;
+var guesses = [];
+var guessLimit = 7;//counts the guesses//
+
+//split the word into an array of letters, that are shown as dashes until guessed correctly//
+var separateWord = function() {
+  word= prompt("Enter a word for your player to guess!"),toLowerCase(); //ask for word//
+  word = word.split("");
+  var placeholders = "" ;
+    for(var i = 0; i < word.length; i+=1) {
+      letters[i]= "_"
+    }
+  }
+  return letters.join("") //convert the dashes back into an array(the word) keeping the dashes instead of the letters//
+}
+//function will continue to ask for more guesses until guess limit is reached. shows the word (split up into an array of dashes, and how many guesses are left)//
+var getGuess= function () {
+  while(numberWrongGuesses < guessLimit) {
+  prompt("Guess a letter.");
+  guesses.filter(word);
+}
+}
+function checkLetter(letter) {
+  guesses.push(letter);
+  var wrongGuess = wrongChoice(letter);
```

Comment:
Doesn't look like you define this `wrongChoice` function either. I would suggest not referencing these undefined variables in your code until you actually define them. Otherwise they're going to cause your app not to work, and that can get particularly messy when there's a lot of them.